### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by splitt3r**
@dartcafe you should be able to contribute here. The first basic unit test is ready so we have a template for further ones. I´m using http://factory-muffin.thephpleague.com/ for the object creation.

And i removed the deprecated hash method [here](https://github.com/splitt3r/polls/blob/08eb3e5efc73e173520890113a5f41329ce7b8f8/lib/Controller/PageController.php#L433-L436) so we should do some test if it´s ready.

### CODE REVIEW

**Comment by dartcafe** [comment]
Hmmm. I have to go into it, but I would say, we should start with the simple tests. At now, I am not able to understand the whole context.

**Comment by dartcafe** [comment]
The main problem is, I am ill at the moment and will be away for a couple of days, So I will not be able to be here frequently.

**Comment by splitt3r** [comment]
No problem. I´m busy atm anyway so take your time.

File: lib/Controller/PageController.php [hunk 1]
```diff
@@ -423,15 +424,16 @@ public function insertPoll(
...
 		$event->setDescription(htmlspecialchars($pollDesc));
 		$event->setOwner($userId);
 		$event->setCreated(date('Y-m-d H:i:s'));
-		$event->setHash(\OC::$server->getSecureRandom()->getMediumStrengthGenerator()->generate(16,
-			ISecureRandom::CHAR_DIGITS .
-			ISecureRandom::CHAR_LOWER .
-			ISecureRandom::CHAR_UPPER));
+		$event->setHash($secureRandom->generate(
+			16,
+			'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
```
Comment (splitt3r) [inline lib/Controller/PageController.php]: https://github.com/pellaeon/registration/commit/01fa314fc57c9e172f54a07320ba5ee5ebbd350c

**Comment by splitt3r** [comment]
Code coverage detection https://scrutinizer-ci.com/g/splitt3r/polls/?branch=unit-tests works now.

**Comment by splitt3r** [comment]
It´s ready for testing / reviewing now 😀

**Pull request merged**